### PR TITLE
LogoutSpots chams render

### PR DESCRIPTION
**Pull request by cattyngmd**

### CODE REVIEW

**Comment by cattyngmd** [comment]
https://github.com/3arthqu4ke/3arthh4ck/issues/153

**Comment by 3arthqu4ke** [comment]
Wow you are fast, give me a sec to review!

Closes #153

**Comment by cattyngmd** [comment]
huh

**Review by 3arthqu4ke — APPROVED** [review]
LGTM! 🔥

**Comment by 3arthqu4ke** [comment]
Thank you!!!

**Comment by cattyngmd** [comment]
np :D

File: src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java [hunk 1]
```diff
@@ -27,23 +28,59 @@ public void invoke(Render3DEvent event)
             {
                 AxisAlignedBB bb = Interpolation.interpolateAxis(spot.getBoundingBox());
                 RenderUtil.startRender();
-                RenderUtil.drawOutline(bb, 1.5f, Color.RED);
+
+                if(module.chams.getValue()) {
```
[the remark this answers is not in the document]
Comment (cattyngmd) [reply src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java:32]: oof

**Pull request merged**